### PR TITLE
Reorder readme, Custom homepage

### PR DESCRIPTION
**Pull request by lovincyrus**
closes https://github.com/allinbits/design/issues/336

Preview: https://deploy-preview-384--sdk-tutorials.netlify.app

- reorder tutorials on sidebar and readme based on difficulty
- fixed ordering of starport-blog (01, 02, etc.)
- custom homepage w/ beginner, intermediate, advanced tags
- custom homepage w/ video section

image
image

## TODO
- [x] Nass to review / design QA.
- [x] Marko / Billy to review the tutorial cards desc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,77 @@
...
+    url: /starport-blog/01-index.html
+    tag: beginner
+  - title: Scavenge
+    desc: Users post questions with hashed answers and bounties for whoever can solve them using a commit-reveal scheme that prevents front-running.
+    url: /scavenge/tutorial/01-background.html
+    tag: intermediate
+  - title: Nameservice
+    desc: Build a fully functional naming service on a blockchain.
+    url: /nameservice/tutorial/00-intro.html
+    tag: advanced
```
Comment (marbar3778) [inline README.md:21]:
```suggestion
    tag: intermediate
```

File: README.md [hunk 2]
```diff
@@ -1,38 +1,77 @@
...
+  - title: Cosmos Burner Chain
+    desc: Provide a blueprint for your own application specific blockchain that interacts with other EVM based networks like Ethereum and xDai.
+    url: /burner-chain/00-index.html
+    tag: advanced
```
Comment (marbar3778) [inline README.md:25]:
```suggestion
    tag: intermediate
```

We don't have any advanced tutorials.

File: .vuepress/components/Home.vue [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+    }
+
+    &:active {
+      transition-duration: 0s;
+      opacity: 0.7;
+    }
+
+    &__tag {
+      color: var(--tag-text-color);
+      font-size: 14px;
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:159]: `0.875rem`

File: .vuepress/components/Home.vue [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+      line-height: 20px;
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:160]: `1.25rem`

File: .vuepress/components/Home.vue [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+      letter-spacing: 0.03em;
+      text-transform: capitalize;
+      background: var(--tag-background-color);
+      border-radius: 4px;
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:164]: `0.25rem`

File: .vuepress/components/Home.vue [hunk 6]
```diff
@@ -0,0 +1,314 @@
...
+      padding: 2px 8px;
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:165]: `0.125rem 0.5rem`

File: .vuepress/components/Home.vue [hunk 7]
```diff
@@ -0,0 +1,314 @@
...
+      width: fit-content;
+    }
+
+    &__title {
+      font-weight: 500;
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:170]: Sadly Figma didn't update the styles from the external library properly (really annoying bug that I keep forgetting about) so you'll have got these type styles from the old Gotham styles - sorry! Here's what it should be:

```
font-weight 600
font-size 1.125rem
line-height 1.6875rem
```

No letter spacing adjustment needed.

File: .vuepress/components/Home.vue [hunk 8]
```diff
@@ -0,0 +1,314 @@
...
+      letter-spacing: -0.01em;
+      color: #000000;
+      margin-bottom: 0.5rem;
+    }
+
+    &__desc {
+      font-size: 0.875rem;
+      line-height: 1.25rem;
+      letter-spacing: 0.03em;
+      color: rgba(0, 0, 0, 0.667);
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:182]: `var(--color-text-dim, inherit)`

File: .vuepress/components/Home.vue [hunk 9]
```diff
@@ -0,0 +1,314 @@
...
+      padding: 2px 8px;
+      width: fit-content;
+    }
+
+    &__title {
+      font-weight: 500;
+      font-size: 1.25rem;
+      line-height: 1.75rem;
+      letter-spacing: -0.01em;
+      color: #000000;
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:174]: `var(--color-text, inherit)`

File: .vuepress/components/Home.vue [hunk 10]
```diff
@@ -0,0 +1,314 @@
...
+}
+
+.stack {
+  display: grid;
+  gap: 1.5rem;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  margin-bottom: 4rem;
+
+  &__item {
+    color: var(--color-text, black);
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:194]: `var(--color-text, inherit)`

File: .vuepress/components/Home.vue [hunk 11]
```diff
@@ -0,0 +1,314 @@
...
+    transition: box-shadow 0.25s ease-out, transform 0.25s ease-out, opacity 0.4s ease-out;
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:195]: Don't need box-shadow or transform transitions here.

File: .vuepress/components/Home.vue [hunk 12]
```diff
@@ -0,0 +1,314 @@
...
+      transition-duration: 0s;
+    }
+
+    &__image {
+      width: 100%;
+      object-fit: cover;
+    }
+
+    &__h1 {
+      font-size: 1.25rem;
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:208]: Let's reduce this size a bit. We can reduce it further when we increase to 4 columns - once we have another video to display.

```
font-size 1rem
line-height 1.5rem
margin-top 0.5rem
margin-bottom 0.25rem
```

File: .vuepress/components/Home.vue [hunk 13]
```diff
@@ -0,0 +1,314 @@
...
+      line-height: 1.5rem;
+      font-weight: 600;
+      margin-top: 12px;
+      margin-bottom: 12px;
+    }
+
+    &__p {
+      font-size: 0.875rem;
+      color: rgba(22, 25, 49, 0.65);
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:217]: `var(--color-text-dim, inherit)`

File: .vuepress/components/Home.vue [hunk 14]
```diff
@@ -0,0 +1,314 @@
...
+    grid-template-columns: repeat(auto-fill, minmax(240px, 1fr));
+    margin-bottom: 3rem;
+
+    &__item {
+      &__text {
+        grid-template-columns: 3rem 1fr;
+        text-align: start;
+      }
+
+      &__h1 {
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:306]: All of this can be removed now that we're reducing the size of the the base styles for this element.

**Review by nassdonald — CHANGES_REQUESTED** [review]
Nice work! Just some small styling tweaks to make, plus the following:

- [ ] Update the sidebar footer links to the following:
image
- [ ] Update the video titles to their titles on YouTube – I know this is less aesthetically pleasing, but it's better to stay accurate here.
  - Cosmos Code With Us - Building your first Cosmos app
  - Getting started with Starport, the easiest way to build a Cosmos SDK blockchain
  - Code with us - Scavenger Hunt Tutorial with Billy Rennekamp
- [ ] Add "IBC Protocol" to the _Documentation_ column in the footer.
- [ ] Remove the padding from the Discord icon in the _Help & Support_ section. Not sure how that got there, it's the same on the Cosmos SDK docs – which also needs to be removed. 😄 
image

File: .vuepress/components/Home.vue [hunk 15]
```diff
@@ -0,0 +1,314 @@
...
+}
+
+.p {
+  font-size: 1.5rem;
+  line-height: 2.25rem;
+
+  &__alt {
+    margin-top: 0.75rem;
+    margin-bottom: 2rem;
+    font-size: 1.25rem;
```
Comment (nassdonald) [inline .vuepress/components/Home.vue:121]: Another failing on my part to update the Figma file with the newer type styles. Let's increase this to:
```
font-size 1.5rem
line-height 2.25rem
```
Then reduce to the sizes you have here @ `(max-width: 1136px)` instead – see the SDK docs homepage for reference.

**Review by nassdonald — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lovincyrus) [reply README.md:21]: wdyt? @okwme

Comment (okwme) [reply README.md:21]: maybe we use advanced until we have something more advanced and then we reduce it?

**Comment by lovincyrus** [comment]
Merge for now. 

**_RE_**: nameservice and cosmos burner chain
We can reduce the level from `advanced` → `intermediate` when we have something more advanced. https://github.com/cosmos/sdk-tutorials/pull/384#discussion_r477596678

**Pull request merged**